### PR TITLE
Style kasten

### PR DESCRIPTION
**Pull request by tlssuse**
@dariavladykina : This document has been updated to use the new TRD stylesheet along with additional content updates.
Please review and, when ready, add to the repo rebuild queue.

### CODE REVIEW

File: kubernetes/start/veeam/adoc/gs_rancher_veeam-kasten.adoc [hunk 1]
```diff
@@ -562,53 +728,57 @@ You can further customize what is and what is not covered under a K10 applicatio
 
 == Working with policies
 
-When you have created a policy and have navigated back to the main dashboard, you will see the selected applications quickly switch from unmanaged to non-compliant.
+When you have created a policy and have navigated back to the main {otherproduct1short} dashboard, you see the selected applications quickly switch from unmanaged to non-compliant.
 That is, a policy covers the objects but no action has been taken yet.
```
Comment (dariavladykina) [inline kubernetes/start/veeam/adoc/gs_rancher_veeam-kasten.adoc:732]:
```suggestion
That is, a policy covers the objects, but no action has been taken yet.
```

File: kubernetes/start/veeam/adoc/gs_rancher_veeam-kasten.adoc [hunk 2]
```diff
@@ -626,29 +796,30 @@ After the restore completes, you can go back to your application and verify that
...
-When the filter is in effect, you see applications that K10 has previously protected but which no longer exist.
+When the filter is in effect, you see applications that {otherproduct1short} had previously protected but which no longer exist.
 These can now be restored using the normal restore workflow.
 
 
 == Summary
 
-SUSE Rancher enables enterprises to streamline multi-cluster Kubernetes operations everywhere with unified security, policy, and user management.  Kasten K10 by Veeam delivers easy-to-use, Kubernetes native application backup and restore, disaster recovery, and application mobility.  Together, SUSE and Veeam provide enterprises with the tools they need to reduce risk and accelerate cloud native success.
+Rancher by SUSE enables enterprises to streamline multi-cluster Kubernetes operations everywhere with unified security, policy, and user management.
+{otherproduct1} delivers easy-to-use, Kubernetes native application backup and restore, disaster recovery, and application mobility.
```
Comment (dariavladykina) [inline kubernetes/start/veeam/adoc/gs_rancher_veeam-kasten.adoc:808]:
```suggestion
Rancher by SUSE enables enterprises to streamline multi-cluster Kubernetes operations everywhere with unified security, policy and user management.
{otherproduct1} delivers easy-to-use, Kubernetes-native application backup and restore, disaster recovery, and application mobility.
```

**Review by dariavladykina — APPROVED** [review]

**Comment by fsundermeyer** [comment]
LGTM - merging.

**Comment by tlssuse** [comment]
Thank you, @dariavladykina and @fsundermeyer !

**Pull request merged**